test/integration/asset-integration.js

```diff
@@ -45,7 +45,7 @@ describe('Asset api', function () {
     let environment
 
     before(async () => {
-      space = await createTestSpace(initClient(), 'Assets')
+      space = await createTestSpace(initClient({ retryOnError: false }), 'Assets')
```

Comment:
Adding this option so that tests don't take unnecessarily long. Otherwise with five retries (with increasing backoff times) it would take about 25 seconds for the new test to fail